core/store/migrations/0022_create_log_config_table.go

```diff
@@ -0,0 +1,70 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+const up22 = `
+		CREATE TYPE service AS ENUM (
```

Comment:
What's with the indentation?

Reply:
saw some conflicting indentation styles in the migrations (0 tab indents, 2 tab indents, 1..) , can go with 1 tab indent if that's preferred, seams cleanest?